src/cmd-compress

```diff
@@ -12,10 +12,21 @@ import argparse
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file, rm_allow_noent
 
+# Hardcoded value for compression algorithm for now
+DEFAULT_COMPRESSION_ALGO = 'xz'
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
+parser.add_argument("--algorithm",
```

Comment:
[`choices`](https://docs.python.org/3.5/library/argparse.html#choices) would be neat here.

Reply:
Updated :+1: